index.html

```diff
@@ -15,6 +15,10 @@ <h1>Justus Massmann</h1>
       <a href="#biography">Biography</a>
     </nav>
   </header>
+  <figure>
+  <img src="https://source.unsplash.com/random/600x300" alt="profile picture">
+    <figcaption>This is my profile picture for my personal website.</figcaption>
+</figure>
```

Comment:
I like your alt description! Please check the indentation of these lines. 👯

Reply:
Oh yeah, you're right! Thank u for the hint :)